src/main/java/duke/commands/CommandBye.java

```diff
@@ -0,0 +1,40 @@
+package duke.commands;
+
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents the "bye" command in the Duke application. This command allows
+ * the user to exit the application.
+ */
+public class CommandBye extends Command {
+
+    /**
+     * Constructs a new CommandBye object.
+     * This constructor is empty as no specific initialization is needed for the "bye" command.
+     */
+    public CommandBye() {}
+
+    /**
+     * Indicates that executing this command should result in exiting the Duke application.
+     *
+     * @return true to indicate an exit command, false otherwise.
+     */
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+
+    /**
+     * Executes the "bye" command, which displays a farewell message to the user.
```

Comment:
For Javadocs overridden methods, maybe you can consider using the `@inheritDoc` tag? The same for the rest of your overridden method Javadocs.

Reply:
Thank you! I will amend accordingly :)